website/layouts/guides.erb

```diff
@@ -1,6 +1,9 @@
 <% wrap_layout :inner do %>
   <% content_for :sidebar do %>
     <ul class="nav docs-sidenav">
+      <li<%= sidebar_current("guides-getting-started") %>>
+        <a href="/intro/getting-started/install.html">Getting Started</a>
+      </li>
```

Comment:
I'd consider doing the same thing in the `docs.erb` layout too... I know I always end up there before I remember that we have two other "documentation" sections in the nav.